### ISSUE TITLE
PNG Mapping support

### ISSUE THREAD

**Issue by bisechen** [f3d-app/f3d#586]
I found that F3D does not seem to support PNG Mapping during use?
Incorrect display colors for png mapping
![UPGpZMXvvJ](https://user-images.githubusercontent.com/18411296/215716614-4eeffc7c-f8f3-40d7-9507-dbf185d5c6ab.png)

**Comment by mwestphal** [comment]
Hi @bisechen 

Could you share the .fbx file ? This may be an issue with assimp, VTK of F3D so we need to test it.

**Comment by mwestphal** [comment]
>  support PNG Mapping during use?

Could you elaborate on that ?

**Comment by Meakk** [comment]
Looks like the colors are mapped. It can happen if the texture is 16-bits (which is possible with PNG files).  
It's probably an easy fix by calling `vtkTexture::SetColorModeToDirectScalars` in the assimp importer.  
Please @bisechen share the file.

**Comment by bisechen** [comment]
> Looks like the colors are mapped. It can happen if the texture is 16-bits (which is possible with PNG files). It's probably an easy fix by calling `vtkTexture::SetColorModeToDirectScalars` in the assimp importer. Please @bisechen share the file.

You are right, I tried it again, it is indeed the problem caused by 16 bit.

**Comment by bisechen** [comment]
> > support PNG Mapping during use?
> 
> Could you elaborate on that ?

Is the problem caused by PNG 16bit

**Comment by bisechen** [comment]
> Still, it should work with 16-bits PNG files. Please share the file so we can fix the issue.

Because the test file has been deleted, I recreated a 16bit mapping using the original settings and steps, but the FBX file cannot be uploaded here, but this FBX is only Blender's default CUBE export
![Cube_Bake1_PBR_Diffuse](https://user-images.githubusercontent.com/18411296/215934158-2d8fc0cf-8eca-4e27-a717-31564f84a471.png)

**Comment by mwestphal** [comment]
Any change to look at this @Meakk ?

**Comment by bisechen** [comment]
> Any change to look at this @Meakk ?

I downloaded version 2.0 for testing, 
and the display of both 16-bit and 32-bit color depth is incorrect, 
but the display of 8-bit color depth is normal.

Previously, I have been using Blender for drawing, 
but in order to rule out its problems, 
this time I used Krita to create files with three different color depths. 
The display results of F3D in 16-bit and 32-bit are still incorrect.

I have attached my test files for reference.
Krita version 5.1.5
Blender version 3.5

[https://drive.google.com/file/d/1FYAzb1pP1henVIrdvJK7JXyXPxR92PkX/view?usp=share_link](url)

**Comment by Meakk** [comment]
Hi @bisechen  
Here's the current result for the 16-bit fbx file:

![old](https://user-images.githubusercontent.com/6069752/236704066-55ade796-852f-4304-b0e1-52d0815c24a6.png)

Here's with my fix:

![fixed](https://user-images.githubusercontent.com/6069752/236704076-aaf0be80-b783-42d6-84d0-7bfd3b29bb52.png)

Is it what you are expecting?

**Comment by bisechen** [comment]
> Hi @bisechen Here's the current result for the 16-bit fbx file:
> 
> ![old](https://user-images.githubusercontent.com/6069752/236704066-55ade796-852f-4304-b0e1-52d0815c24a6.png)
> 
> Here's with my fix:
> 
> ![fixed](https://user-images.githubusercontent.com/6069752/236704076-aaf0be80-b783-42d6-84d0-7bfd3b29bb52.png)
> 
> Is it what you are expecting?

Yes, this is the result I want. Thanks for your contribution.
